praw/models/inbox.py

```diff
@@ -80,6 +80,23 @@ def comment_replies(
             self._reddit, API_PATH["comment_replies"], **generator_kwargs
         )
 
+    def mark_all_read(self):
+        """Mark all messages as read with just one API call.
+
+        Example usage:
+
+        .. code-block:: python
+
+            reddit.inbox.mark_all_read()
+
+        .. note::
+
+            This method returns after Reddit acknowleges your request, instead of after
+            the request has been fulfilled.
```

Comment:
Are you saying that marking the messages as read takes a bit if you have a lot of unread messages?

Reply:
The docs [say](https://www.reddit.com/dev/api#POST_api_read_all_messages).

    Queue up marking all messages for a user as read.
    
    This may take some time, and returns 202 to acknowledge acceptance of the request.

You could potentially have thousands, hundreds or thousands, or even millions of unread messages if you are a regular submitter to a high volume sub and you leave inbox replies on. I don't know what the actual upper limit is.  It would take some time to mark all of those as unread, so they just return a 202.  I could go through my inbox and mark everything as unread to test this, but I'm inclined to take their word for it, since it makes sense.

Reply:
Ah I see. The note just had me confused for a sec, I might just have read it wrong.